utils/check_codeblocks.sh

```diff
@@ -26,7 +26,7 @@ do
   fi
 done
 
-for FILE in $(find tests/src/ -type f -name "*.h" -o -name "*.cpp" | sed s,^tests/src/,, | sort)
+for FILE in $(find tests/unit/ -type f -name "*.h" -o -name "*.cpp" | grep -v -e "^tests/unit/include" | sed s,^tests/unit/src,, | sort)
```

Comment:
You don't need the additional grep filter if you restrict searching the first find to `tests/unit/src/`.